src/components/filter/FilterDropDown.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import * as Styled from "./FilterDropDown.styled";
+
+interface FilterDropDownProps {
+  id: string;
+  labelValue: string;
+  content: any;
+  handleChange: Function;
+}
+
+function FilterDropDown({ id, labelValue, content, handleChange }: FilterDropDownProps) {
+  const options = new Set<string>();
+
+  content?.map((item) => options.add(item[id]));
+  const optionsArray = Array.from(options);
```

Comment:
Nice! Would be even better if you can filter out the undefined values :)

Reply:
Good point!